Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -221,6 +268,14 @@ class AuthenticationSettingsViewController: SettingsTableViewController {
             navigationItem.title = AuthenticationStrings.passcode
         }
     }
+    
+    private var authenticationSectionTitle: NSAttributedString {
+        if LAContext().canEvaluatePolicy(.DeviceOwnerAuthenticationWithBiometrics, error: nil) {
+            return NSAttributedString(string: AuthenticationStrings.authenticateFeaturesWithTouchIDPasscode)
+        } else {
+            return NSAttributedString(string: AuthenticationStrings.authenticateFeaturesWithPasscode)
+        }
```

Comment:
She thinks it would be more consistent with existing patterns to stick with the current scheme: https://bugzilla.mozilla.org/show_bug.cgi?id=1256470#c7